flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/job/JobDetailsInfo.java

```diff
@@ -50,7 +50,7 @@
 
 	public static final String FIELD_NAME_JOB_NAME = "name";
 
-	public static final String FIELD_NAME_IS_STOPPABLE = "isStoppable";
+	public static final String FIELD_NAME_IS_STOPPABLE = "stoppable";
```

Comment:
We should figure out why it returned "stoppable", as far as i can tell it should "isStoppable". This would be an API breaking change.